Packages/MIES/MIES_IVSCC.ipf

```diff
@@ -367,6 +367,16 @@ Function IVS_RunGigOhmSealQC()
 	CtrlNamedBackground IVS_finishGigOhmSealQCCheck, start
 End
 
+// @brief Loads a single stimulus for the user when using the ZMQ Proxy
```

Comment:
@aleonlein missing forward slash

Reply:
@timjarsky Right. But unfortunately I was too eager to merge it. I'll fix that.

Reply:
@timjarsky  good catch - I will fix and resubmit.

Reply:
@aleonlein I've already fixed it in 2471.